src/main/scala/scorex/api/http/AddressApiRoute.scala

```diff
@@ -228,6 +238,19 @@ case class AddressApiRoute(settings: RestAPISettings, wallet: Wallet, state: Sta
       .getOrElse(InvalidAddress)
   }
 
+  private def balancesDetailsJson(address: String)(implicit transactionModule: TransactionModule): ToResponseMarshallable = {
```

Comment:
please make this method of tansaction module with sig : `String=>BalanceDetails`, than invoke it here. it should be marshalled automatically